java/openjdk10/Portfile

```diff
@@ -38,11 +38,13 @@ build {}
 destroot.violate_mtree yes
 
 destroot {
+    ui_msg "\nWarning: Support for OpenJDK 10.x has reached end of life and there will be no more updates. Consider upgrading to a supported OpenJDK version.\n"
```

Comment:
My only comment: this should break the line at 80 columns or less. (Maybe two `ui_msg` calls?

Otherwise this is perfect. Thank you.